retrieve_09S2y000000iMwPEAU/classes/StudentDetailsController.cls

```diff
@@ -8,9 +8,8 @@ public class StudentController
     public StudentController( )
     {
         
-        //Id studentId = Apexpages.CurrentPage().getParameters().get('Id');
-        //System.debug(studentId);
-        myStudent = [SELECT Id, Class__c, Name, DOB__c, Class__r.Name, email__c FROM Student__c LIMIT 1 ];
+        Id studentId = Apexpages.CurrentPage().getParameters().get('Id');
+        myStudent = [SELECT Id, Class__c, Name, DOB__c, Class__r.Name, email__c FROM Student__c WHERE Id =: studentId LIMIT 1 ][0];
```

Comment:
we don't really need the LIMIT here as it will always return single result for `Id = ` as id is unique